storey/table.py

```diff
@@ -977,11 +980,13 @@ def aggregate(self, time, value):
 class LastValue(AggregationValue):
     name = 'last'
 
-    def __init__(self, max_value=None, set_data=None):
-        super().__init__(max_value, set_data)
+    def __init__(self, max_value=None, set_data=None, set_time=None):
```

Comment:
not related to __init__
LastValue doesn't have reset() impl of it's own so it will use the one in AggregationValue and will set set_time to None instead to -math.inf